R/module_wunder_bar.R

```diff
@@ -0,0 +1,112 @@
+#' Manager bar module
+#'
+#' Bar of buttons that open modal dialogs.
+#'
+#' Creates a bar of buttons that open modal dialogs where manager modules reside.
+#' Currently contains three modules:
+#' - [`module_filter_manager`]
+#' - [`module_snapshot_manager`]
+#' - [`module_bookmark_manager`]
+#'
+#' The bar is placed in the `teal` app UI, next to the filter panel hamburger.
+#'
+#' @name module_wunder_bar
+#' @aliases wunder_bar wunder_bar_module
+#'
+#' @param id (`character(1)`) `shiny` module instance id.
+#' @inheritParams module_filter_manager
+#'
+#' @return Nothing is returned.
+
+#' @rdname module_wunder_bar
+#' @keywords internal
+wunder_bar_ui <- function(id) {
```

Comment:
just curious: what does "wunder bar" mean in this context? :)

Reply:
It's just a wonderul bar :wink: